doc/python/domdec.rst

```diff
@@ -12,7 +12,9 @@ Load balancers
 
 Load balancing generates a :class:`domain_decomposition` given an :class:`arbor.recipe`
 and a description of the hardware on which the model will run. Currently Arbor provides
-one load balancer, :func:`partition_load_balance`, and more will be added over time.
+one load balancer, :func:`partition_load_balance`; and a function for creating
+custom decompositions, :func:`partition_by_group`.
+More load balancers will be added over time.
```

Comment:
`may` rather than `will`?

Reply:
I think we need to add more, such as your suggested random partitioning, or round-robin. This is us acknowledging that the available functions aren't sufficient. That being said, I can change it to may.